### PR TITLE
Add September release notes

### PR DESCRIPTION
**Pull request by thomasp-ms**
Adding the September release notes. After the iteration 2 branch gets merged into main, we'll create a release using these notes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 
-*Features*
-* ...
+We are excited to announce the release of the September iteration of our [FL Accelerator repository](https://github.com/Azure-Samples/azure-ml-federated-learning).
 
-*Bug Fixes*
-* ...
+Here are the new features.
+- [Single location](https://github.com/Azure-Samples/azure-ml-federated-learning/blob/release-sdkv2-iteration-02/README.md) for all documentation.
+- Refactored repository - removed redundant directories/files.
```
Comment (jfomhover) [inline CHANGELOG.md:9]: "cleaned up" instead of refactored?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+- Auto-provisioning tools now provision a setup with _secured_ internal silos (using Bicep templates), with extra VNet protection to prevent from exfiltrating data during an FL job. Connect your own storage accounts to this setup, and easily run FL jobs on your own data!
```
Comment (jfomhover) [inline CHANGELOG.md:10]: proposed rewording:
- Auto-provisioning scripts to create a complete sandbox using bicep templates, by default an open sandbox on 3 regions, an experimental script with vnets and private endpoints is also available.
- A quickstart to run an FL demo in minutes, and easily run FL jobs on your own data!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,21 @@
-## [project-title] Changelog
+# FL Accelerator Changelog
 
-<a name="x.y.z"></a>
-# x.y.z (yyyy-mm-dd)
+##  September 2022 release
```
Comment (jfomhover) [inline CHANGELOG.md:3]: Could we have sub sections like

### Repository Structure
- ...

### Provisioning
- ...

### Documentation
- ...

### FL Experience
(put metrics + factory here)

**Review by jfomhover — CHANGES_REQUESTED** [review]

**Comment by thomasp-ms** [comment]
Changes done @jfomhover

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,13 +1,30 @@
...
+
+### FL Experience
+- Easy navigation using Azure Storage Explorer to discover models of various iterations, here is a more intuitive path (\<experiment-name\>/\<iteration-num\>/\<model-name\>)
+- Introduction of "factory" code to simplify the creation of FL experiments.
+  - Decoupled components for more readability.
+  - Soft validation to check the correct permissions of your assets.
+- Metrics:
+  - The combined losses and model performances of several silos can now be seen at the pipeline level.
+
+To get started, go [here](https://github.com/Azure-Samples/azure-ml-federated-learning/blob/main/docs/quickstart.md)!
```
Comment (jfomhover) [inline CHANGELOG.md:28]: Shouldn't we use relative links here? like ./docs/quickstart.md ?

**Review by jfomhover — APPROVED** [review]
Looks good thanks! Just a minor detail about the url to consider.

File: CHANGELOG.md (hunk 4 above)
Comment (thomasp-ms) [reply CHANGELOG.md:28]: oops, you're right

**Pull request merged**